es6-classes/circle.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Square {
+  constructor(area, width, circumference, radius) {
```

Comment:
Per instruction:
Write its constructor to take the **circle's radius as a parameter** (and nothing else)